packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,400 @@
+---
+title: Design Guidelines for Paste
+description: Notes on how to use Paste design assets
+slug: /getting-started/design/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Card} from '@twilio-paste/card';
+import {Tooltip} from '@twilio-paste/tooltip';
+import {Flex} from '@twilio-paste/flex';
+import {version as typesVersion} from '@twilio-paste/types/package.json';
+import {version as stylePropsVersion} from '@twilio-paste/style-props/package.json';
+import {version as designTokensVersion} from '@twilio-paste/design-tokens/package.json';
+import {version as themeVersion} from '@twilio-paste/theme/package.json';
+import {version as coreVersion, dependencies as coreDependencies} from '@twilio-paste/core/package.json';
+import {version as iconsVersion} from '@twilio-paste/icons/package.json';
+import {version as animationVersion} from '@twilio-paste/animation-library/package.json';
+import {version as stylingLibraryPropsVersion} from '@twilio-paste/styling-library/package.json';
+import {version as dropdownVersion} from '@twilio-paste/dropdown-library/package.json';
+import {version as reakitVersion} from '@twilio-paste/reakit-library/package.json';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {InlineCode} from '../../../components/Typography';
+import Package from '../../../../../../package.json';
+
+import { SuccessIcon } from "@twilio-paste/icons/esm/SuccessIcon";
+import { WarningIcon } from "@twilio-paste/icons/esm/WarningIcon";
+import { ErrorIcon } from "@twilio-paste/icons/esm/ErrorIcon";
+
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/getting-started/design/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Installation
+
+You’ll need to download a few things before you can get started using Paste in
```

Comment:
replace "design work" with something around the lines of designing UIs with Paste